draft-ietf-webtrans-http3.md

```diff
@@ -226,7 +226,10 @@ session have alternative mechanisms:
 
 WebTransport over HTTP/3 provides the following features described in
 [OVERVIEW]: unidirectional streams, bidirectional streams and datagrams,
-initiated by either endpoint.
+initiated by either endpoint. Protocols designed for use with WebTransport over
+HTTP/3 are constrained to these features.  The Capsule Protocol is an
+implementation detail of WebTransport over HTTP/3 and is not a WebTransport
```

Comment:
This should probably clarify that sending/receiving new capsules by themselves is not a feature.